lib/sycamore/sycamore/transforms/merge_elements.py

```diff
@@ -611,6 +611,192 @@ def postprocess_element(self, elem: Element) -> Element:
         return elem
 
 
+class HeaderAugmenterMerger(ElementMerger):
+    """
+    The ``HeaderAugmenterMerger`` groups together different elements in a Document and enhances the text
+    representation of the elements by adding the preceeding section-header/title.
+
+    - It merges elements ("Text", "List-item", "Caption", "Footnote", "Formula" , "Page-footer", "Page-header").
```

Comment:
Maybe you want to remove the extra space in `..."Formula" , "Page-footer"...`

Reply:
done